### DISCUSSION TITLE
[Feat]: MIT License

### DISCUSSION THREAD

**Discussion by chenhuanguang** [coder/code-server#5727] [Q&A]
I read your LICENSE description, but I don't quite understand. I am using the code server package now. I have modified the code in it because it is an editor. The open project contains the source code of your package. Do we just need to keep the LICENSE file in the code server directory for commercial use

**Comment by jsjoeio** [comment]
Yes, it should be included: https://github.com/coder/code-server/blob/main/LICENSE#L12-L13
